src/container/BooksList.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] It would be better if we resolve the ESLint warnings instead of disabling it. Fixing these warnings or errors helps us write clean and readable code